java_events/pom.xml

```diff
@@ -0,0 +1,26 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>cpp-event-simple-tutorial</groupId>
+  <artifactId>java-filter</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+    <build>
+    <finalName>java-filters-converters</finalName>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.5.1</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+  	<dependency>
+  		<groupId>org.infinispan</groupId>
+  		<artifactId>infinispan-core</artifactId>
+  		<version>9.0.0.Beta2</version>
```

Comment:
Could you define a property version.org.infinispan, similar to java_tests/pom.xml ? This way we'll be able to override the version from command line.